kordex/build.gradle.kts

```diff
@@ -48,12 +42,16 @@ gradlePlugin {
 val targetAttribute = Attribute.of("org.jetbrains.kotlin.platform.type", String::class.java)
 
 dependencies {
+    shadow(gradleApi())
```

Comment:
FlexVer is actually the only thing I needed to shade in - I guess using shadow on the others should work, right?

Reply:
Imo the shadow config is a bit of a misnomer. It adds dependencies that should not get shadowed, I tried to only shadow FlexVer but it behaved badly, maybe I should add it to the shadow config

Reply:
Yeah because of the way the publish plugin handles shadow, shadowing only one dependency doesn't work